doc/api/configuration.md

```diff
@@ -259,12 +259,17 @@ res = %sql SELECT * FROM languages
 ```{versionadded} 0.9
```

Comment:
Please add a `versionchanged` also since we are modifying it. @edublancas Is there anything else to be done since it looks like a breaking change?

Reply:
[Added version changed](https://github.com/ploomber/jupysql/pull/980/files#diff-7c559917b46c06e53b27fa5152bf29cb1db41f04c53f857ef5331f7a003204ffR259) for now. Can add deprecation warning if we feel its needed

Reply:
Yes please add the warning . You can find an example here:

https://github.com/ploomber/jupysql/blob/062d5e9155f2ec99f8c05baf37b882681d3322b6/src/sql/magic.py#L559

Reply:
@bryannho has the warning been added?

Reply:
@neelasha23 I was able to add a warning when the config file is initially loaded, but have been having trouble showing the warning in cases where `%config` is used. The logic for parsing/warning of `%config` statements is handled by a different library, not JupySQL, which is why it's difficult. Will try to find a workaround

Reply:
when users execute `%config`, we can set custom hooks (and that's where you can print a warning), we have an example already, check the `_mutex_autopandas_autopolars` function.

if that doesn't work, let me know